test/tests.ts

```diff
@@ -228,16 +228,22 @@ export function testNitro(
     _handler = await getHandler();
   }, 25_000);
 
-  it("API Works", async () => {
-    const { data: helloData } = await callHandler({ url: "/api/hello" });
-    expect(helloData).to.toMatchObject({ message: "Hello API" });
+  // netlify-edge intentionally ignores prerendered routes
+  it.skipIf(ctx.preset === "netlify-edge")(
+    "prerendered API routes work",
```

Comment:
@pi0 I believe you didn't like this in the original PR. Let me know if you have any other ideas 🤷🏼.

Reply:
prerendered routes are basically static assets. Maybe worth to implement small stub in request handler that emulates netlify edge behavior? (otherwise current hotfix if allows to move it forward faster is good enough IMO -- we can do later)